src/App.jsx

```diff
@@ -0,0 +1,9 @@
+function App() {
+  return (
+    <div>
+      <h1>Hello world</h1>
```

Comment:
Should this be a p tag? H1 might be too big.

Reply:
I think you are right already, done.